### ISSUE TITLE
Newsletter design

### ISSUE THREAD

**Issue by gregoiredierendonck** [lesintegristes/lesintegristes-theme#42]

**Comment by bpierre** [comment]
:+1:

**Comment by gregoiredierendonck** [comment]
Is there some specific constraints to doing it?

**Comment by bpierre** [comment]
Not really, but we have to use Mailchimp (or a custom system) instead of Feedburner, otherwise we can not change the newsletter template. See #43

**Comment by gregoiredierendonck** [comment]
I've seen it... :)

Ok, adding to "to-do list" !

**Comment by bpierre** [comment]
( When I write “see #num”,  it’s mostly to reference an issue into another issue – GitHub adds a reference on the other issue. I know you saw it :-) )

**Comment by gregoiredierendonck** [comment]
Oups :)
